File: Java/src/Iniciante/Solucao_1035.java
package Iniciante;

import java.util.Scanner;

public class Solucao_1035 {
    public static boolean ehPar(int num) {
        return num % 2 == 0;
    }

    public static boolean ehMaior(int num1, int num2) {
        return num1 > num2;
    }

    public static boolean somaABMaiorQueCD(int a, int b, int c, int d) {
        return c + d > a + b;
    }

    public static boolean ehPositivoCD(int c, int d) {
        return c >= 0 && d >= 0;
    }

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int a, b, c, d;

        a = input.nextInt();
        b = input.nextInt();
        c = input.nextInt();
        d = input.nextInt();

        if (ehMaior(b, c) && ehMaior(d, a) && somaABMaiorQueCD(a, b, c, d) && ehPositivoCD(c, d) && ehPar(a)) {
            System.out.println("Valores aceitos");
        } else {
            System.out.println("Valores não aceitos");
        }
        input.close();
    }
}

File: Python/Iniciante/Solucao_1035.py
a, b, c, d = map(int, input().split())

def eh_par(num):
    return num % 2 == 0

def eh_maior(num1, num2):
    return num1 > num2

def soma_a_b_maior_que_c_d(a, b, c, d):
    return  c + d > a + b

def eh_positivo_c_d(c,d):
    return c >= 0 and d >= 0

if eh_maior(b,c) and eh_maior(d,a) and soma_a_b_maior_que_c_d(a,b,c,d) and eh_positivo_c_d(c,d) and eh_par(a):
    print("Valores aceitos")
else:
    print("Valores nao aceitos")